src/reducers/versions.spec.tsx

```diff
@@ -892,7 +892,10 @@ describe(__filename, () => {
       firstHunk.changes.forEach((change, index) => {
         const externalChange = firstExternalHunk.changes[index];
 
-        expect(change).toHaveProperty('content', externalChange.content);
+        expect(change).toHaveProperty(
+          'content',
+          externalChange.content.replace(/\n$/, ''),
```

Comment:
Do we need to worry about Windows line endings?

Reply:
I am not sure. The parser used by `react-diff-view` only splits on `\n` (DOS is `\r\n` anyway, no?)